tests/unit/local/lambdafn/test_runtime.py

```diff
@@ -72,11 +68,11 @@ def test_must_run_container_and_wait_for_logs(self, LambdaContainerMock):
         # Make sure the container is created with proper values
         LambdaContainerMock.assert_called_with(self.lang, self.handler, code_dir,
                                                memory_mb=self.DEFAULT_MEMORY, env_vars=self.env_var_value,
-                                               debug_port=debug_port, debug_args=debug_arg)
+                                               debug_options=None)
 
         # Run the container and get results
         self.manager_mock.run.assert_called_with(container)
-        self.runtime._configure_interrupt.assert_called_with(self.name, self.DEFAULT_TIMEOUT, container, True)
+        self.runtime._configure_interrupt.assert_called_with(self.name, self.DEFAULT_TIMEOUT, container, False)
```

Comment:
This change actually causes a loss of coverage. We do want to verify that when DebuggingContext is set, the runtime passes True here. 

This is important because the interrupt handler performs something very different when this property is set to True. This is the closest we have to ensuring the debugging continues to work.

Can you set DebugContext object when creating the `LambdaRuntime` object so this condition will hold true?